slack-java-client/src/main/java/com/hubspot/slack/client/SlackWebClient.java

```diff
@@ -716,6 +718,45 @@ protected Long getInitialOffset() {
     return postSlackCommand(SlackMethods.conversations_open, params, ConversationsOpenResponse.class);
   }
 
+  @Override
+  public Iterable<CompletableFuture<Result<List<String>, SlackError>>> getConversationMembers(ConversationMemberParams params) {
+    return new AbstractPagedIterable<Result<List<String>, SlackError>, String>() {
+      @Override
+      protected String getInitialOffset() {
+        return null;
+      }
+
+      @Override
+      protected LazyLoadingPage<Result<List<String>, SlackError>, String> getPage(String offset) throws Exception {
+        if (LOG.isTraceEnabled()) {
+          LOG.trace("Fetching slack usergroup page from {}", offset);
+        }
+
+        CompletableFuture<Result<ConversationMemberResponse, SlackError>> resultFuture = postSlackCommand(
+                SlackMethods.conversations_members,
+                ConversationMemberParams.builder()
+                        .from(params)
```

Comment:
We'll want to set the `cursor` to the `offset` if it exists. Otherwise, we'll always grab the first page

Reply:
I hope I undestood correctly this